src/applications/survivors-benefits/config/chapters/04-household-information/dependentsPages.js

```diff
@@ -0,0 +1,457 @@
+          hideIf: (formData, index) => {
+            const item = formData?.dependents?.[index];
+            const currentPageData = formData;
+            return !(item?.bornOutsideUS || currentPageData?.bornOutsideUS);
+          },
+          labels: COUNTRY_VALUES.reduce((acc, value, idx) => {
+            acc[value] = COUNTRY_NAMES[idx];
+            return acc;
+          }, {}),
+        },
+        'ui:errorMessages': {
+          required: 'Please select a country',
+        },
+      },
+    },
+  },
+  schema: {
+    type: 'object',
+    required: ['birthPlace', 'dateOfBirth'],
+    properties: {
+      dateOfBirth: currentOrPastDateSchema,
+      bornOutsideUS: checkboxSchema,
+      birthPlace: {
+        type: 'object',
+        required: ['city'],
+        properties: {
+          city: { type: 'string' },
+          state: {
+            type: 'string',
+            enum: STATE_VALUES,
+            enumNames: STATE_NAMES,
+          },
+          country: {
+            type: 'string',
+            enum: COUNTRY_VALUES,
+            enumNames: COUNTRY_NAMES,
+          },
+        },
+      },
+    },
+  },
+};
+
+const relationshipPage = {
+  uiSchema: {
+    ...arrayBuilderItemSubsequentPageTitleUI('Relationship to dependent'),
+    relationship: radioUI({
+      title: "What's your relationship to the child?",
+      labels: {
+        BIOLOGICAL: "They're my biological child",
+        ADOPTED: "They're my adopted child",
+        STEPCHILD: "They're my stepchild",
+      },
+    }),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      relationship: radioSchema(['BIOLOGICAL', 'ADOPTED', 'STEPCHILD']),
+    },
+    required: ['relationship'],
+  },
+};
+
+const dependentInfoPage = {
+  uiSchema: {
+    ...arrayBuilderItemSubsequentPageTitleUI("Dependent's information"),
+    inSchool: yesNoUI({
+      title: 'Is your child 18-23 years old and still in school?',
+    }),
+    seriouslyDisabled: yesNoUI({ title: 'Is your child seriously disabled?' }),
+    seriouslyDisabledInfo: {
+      'ui:description': () => <div>{seriouslyDisabledDescription}</div>,
+    },
+    hasBeenMarried: yesNoUI({ title: 'Has your child been married?' }),
+    currentlyMarried: {
+      ...yesNoUI({ title: 'Are they currently married?' }),
+      'ui:options': {
+        expandUnder: 'hasBeenMarried',
+        expandUnderCondition: true,
+      },
+    },
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      inSchool: yesNoSchema,
+      seriouslyDisabled: yesNoSchema,
+      seriouslyDisabledInfo: {
+        type: 'object',
+        properties: {},
+      },
+      hasBeenMarried: yesNoSchema,
+      currentlyMarried: yesNoSchema,
+    },
+    required: ['inSchool', 'seriouslyDisabled', 'hasBeenMarried'],
+  },
+};
+
+const householdPage = {
+  uiSchema: {
+    ...arrayBuilderItemSubsequentPageTitleUI("Dependent's household"),
+    livesWithYou: yesNoUI({ title: 'Does your child live with you?' }),
+    form21Alert: {
```

Comment:
This isn't working like the form alerts in other sections, I can't close or open it. Can you extend the form alert used in other pages like Other income in chapter 6?